contrib/platform/src/com/sun/jna/platform/win32/Winioctl.java

```diff
@@ -72,4 +95,320 @@ public STORAGE_DEVICE_NUMBER(Pointer memory) {
             return FIELDS;
         }
     }
+
+    /**
+     * Indicates the properties of a storage device or adapter to retrieve as the input buffer passed to the
+     * IOCTL_STORAGE_QUERY_PROPERTY control code.
+     */
+    class STORAGE_PROPERTY_QUERY extends Structure {
+
+        public static final List<String> FIELDS = createFieldsOrder("propertyId", "queryType", "additionalParameters");
+
+        /**
+         * Indicates whether the caller is requesting a device descriptor, an adapter descriptor, a write cache
+         * property, a device unique ID (DUID), or the device identifiers
+         * provided in the device's SCSI vital product data (VPD) page.
+         */
+        public /* STORAGE_PROPERTY_ID */ WinDef.DWORD propertyId;
+
+        /**
+         * Contains flags indicating the type of query to be performed as enumerated by the STORAGE_QUERY_TYPE
+         * enumeration.
+         */
+        public /* STORAGE_QUERY_TYPE */ WinDef.DWORD queryType;
+
+        /**
+         * Contains an array of bytes that can be used to retrieve additional parameters for specific queries.
+         */
+        public byte[] additionalParameters = new byte[1];
+
+        public static class ByReference extends STORAGE_PROPERTY_QUERY implements Structure.ByReference {
+            public ByReference() {
+            }
```

Comment:
Don't leave this empty - add call to (implicit) _super()_ as this enables setting a breakpoint in the constructor